requirements/dev.txt

```diff
@@ -10,66 +10,69 @@ astroid==2.3.3
 attrs==19.3.0
 backports.functools-lru-cache==1.6.1
 caniusepython3==7.2.0
-certifi==2019.11.28
+certifi==2020.4.5.1
 chardet==3.0.4
 click-log==0.3.2
-click==7.0
-codecov==2.0.15
-coverage==5.0.3
-ddt==1.3.1                # via -r requirements/quality.txt
-diff-cover==2.6.0
+click==7.1.1
+codecov==2.0.22
+coverage==5.1
+ddt==1.3.1
+diff-cover==2.6.1
 distlib==0.3.0
-django-waffle==0.19.0
-django==2.2.10
+django-waffle==0.20.0
+django==2.2.12
 edx-i18n-tools==0.5.0
 edx-lint==1.4.1
 filelock==3.0.12
-idna==2.8
-importlib-metadata==1.5.0
-inflect==4.1.0            # via jinja2-pluralize
+idna==2.9
+importlib-metadata==1.6.0
+importlib-resources==1.4.0
+inflect==3.0.2            # via jinja2-pluralize
 isort==4.3.21
 jinja2-pluralize==0.3.0   # via diff-cover
-jinja2==2.11.1            # via diff-cover, jinja2-pluralize
+jinja2==2.11.2            # via diff-cover, jinja2-pluralize
 lazy-object-proxy==1.4.3
 markupsafe==1.1.1         # via jinja2
 mccabe==0.6.1
 mock==3.0.5
 more-itertools==8.2.0
-newrelic==5.6.0.135
-packaging==20.1
+newrelic==5.12.0.140
+packaging==20.3
 path.py==12.4.0           # via edx-i18n-tools
 path==13.1.0              # via path.py
-pip-tools==4.4.1
+pathlib2==2.3.5
+pip-tools==5.0.0
```

Comment:
You should add constraint on pip-tools. 
`# pip20+ requires for greater version of pip-tools. But installing 19.3.1 in all our containers and VMs.`
`pip-tools==4.5.1`

Reply:
I don’t think the pip-tools constraint is needed any more for libraries. I think it is needed for IDAs. You can remove this constraint.